packages/client/e2e/_utils/mpe-e2e-utils.ts

```diff
@@ -128,6 +128,16 @@ export function withinMyProfilePageContainer(selector: string): string {
     return `css=[data-testid="my-profile-settings-page-container"] >> ${selector}`;
 }
 
+export function withinUserProfileScreen({
```

Comment:
**🎯 suggestion:** I understand that this method returns a locator. I think this great as it allow a better syntax further in the tests
But might the name change from previous usage ?
Just below `withinMyUserProfilePageContainer` returns the same selector as a string :thinking:

Reply:
In fact I don't understand the name of `withinMyUserProfilePageContainer` function, as it does not return _my_ profile page, but the page of another user. I did not want to change `withinMyUserProfilePageContainer` behavior, so I duplicated it.

Reply:
Ah yes :open_mouth: didn't read the name carefully
I guess it's a typo :thinking: 
Then no big deal we'll see later if we wanna change that